client/landing/stepper/declarative-flow/internals/steps-repository/components/migration-survey/index.tsx

```diff
@@ -11,17 +13,22 @@ const linkByCountry = {
 
 type Countries = keyof typeof linkByCountry;
 
-type MigrationSurveyProps = Pick< SurveyProps, 'isOpen' > & {
-	countryCode: string;
-};
+type MigrationSurveyProps = Pick< SurveyProps, 'isOpen' >;
```

Comment:
Not directly related, but it's unclear to me if/where `isOpen` is being passed to this component.

Reply:
The `isOpen` prop is there all the time since #95408 when the entire `Survey` component was added, and it was never used -- always defaults to `true`. The `AsyncMigrationSurvey` doesn't pass it. @gabrielcaires (the author) could know more about the intent. Currently it seems the prop can be safely removed.

Reply:
It looks like `isOpen` exists across a file different files, so it might save those changes for a quick follow-on pull request.